src/perf_analyzer.cc

```diff
@@ -242,7 +242,7 @@ PerfAnalyzer::CreateAnalyzerObjects()
         pa::RequestRateManager::Create(
             params_->async, params_->streaming, params_->measurement_window_ms,
             params_->max_trials, params_->request_distribution,
-            params_->batch_size, params_->max_threads,
+            params_->schedule, params_->batch_size, params_->max_threads,
```

Comment:
IMO, this should be a new load manager, could be a subclass of RequestRateManager, which takes a schedule.

We already have CustomLoadManager, which takes a text file with intervals.

This PR is like CustomLoadManager but instead of a file, it takes the times on the command line, and instead of _intervals_ (i.e. time _between_ request sends), it takes absolute timestamps, meaning like

`--schedule=1,2,4.5`

sends a request at 1 second after the start of the benchmark, 2 seconds after benchmark start, and 4.5 seconds after benchmark start.